### DISCUSSION TITLE
New Password and Vendors Modules Relation

### DISCUSSION THREAD

**Discussion by antcorro** [YetiForceCompany/YetiForceCRM#15678] [How to] [answered]
Hello,

I have installed YetiForce 6.3.0 SecurityFix v6. Before upgrading to 6.3, the old Password module has a relation with Vendors module, so that from a Vendor detail I saw the related Password tab and as an Admin I was able to configure it from "Modules - Relations" page (see attach)
 
![Screenshot 2022-01-31 at 00-11-42 Vendors Google](https://user-images.githubusercontent.com/10244840/151722688-2e180d1d-826a-44b0-b804-8243fbf93dc4.png)

![Screenshot 2022-01-31 at 00-13-20 Layout Editor](https://user-images.githubusercontent.com/10244840/151722693-6a21276a-b622-49e6-a2a8-253f712c3d3b.png)

After upgrading to v6.3.0 and activating the new Password module, I see that there is no more relation. There is no "Password" tab in vendor detail page.

I have checked the online gitstable version (https://gitstable.yetiforce.com) and I have notice that in this case it's possible to add new relation from "Modules - Relations" page, despite the versions are the same.

![Screenshot 2022-01-31 at 00-41-51 Layout Editor](https://user-images.githubusercontent.com/10244840/151722828-8660e1af-b94c-45a1-b0d8-ee250ca7a9b9.png)

Two Questions:
1) How can I add the Passwords (new module) tab in Vendor detail page?

2) How can I enable the "Add relation" button in "Modules - Relations" page?

Thank you

**Comment by rskrzypczak** [comment, marked as answer]
> How can I add the Passwords (new module) tab in Vendor detail page?

You can't add this from the GUI.
Here you have the fix: https://github.com/YetiForceCompany/YetiForceCRM/commit/eff41081c5423cdf022e5137a2489b4fdd0857d5#diff-6c1ecf2857599d07dbe8f7fbaaeae68350e57e60c25afa88bc62648e30cbc804R13740
You can add it yourself to the database or wait for version 6.4 to be released.

> How can I enable the "Add relation" button in "Modules - Relations" page?

I sincerely advise against modifying relationships. You can mess up a lot here, leading to data errors.
So the manipulation of relationships should be done with full awareness.
This button can be activated in the configuration file: https://doc.yetiforce.com/code/classes/Config-Developer.html#property_CHANGE_RELATIONS

**Reply by antcorro** [reply]
Thankyou very much. I'll wait for 6.4

**Comment by antcorro** [comment]
Hello,
i waited for version 6.4, but after the update all module relationships disappeared. For example, in the Vendor details (like screenshot above) there are no more tabs "Activities", "Products", "Contacts", "Documents" .... What happens?

![Screenshot 2022-09-30 at 19-00-20 Vendors Google](https://user-images.githubusercontent.com/10244840/193320659-de0d68f1-4b2d-4b98-8ab2-e805bae222d1.png)

The same for all modules, like Accounts, Sales Invoices, Products and so on.
